avm/utilities/tools/Invoke-WorkflowsForBranch.ps1

```diff
@@ -118,26 +147,46 @@ function Get-GitHubModuleWorkflowList {
         [string] $RepositoryName,
 
         [Parameter(Mandatory = $false)]
-        [string] $Filter = 'avm.res.*'
+        [switch] $IncludeDisabled,
+
+        [Parameter(Mandatory = $false)]
+        [string] $Filter = 'avm\.(?:res|ptn)'
```

Comment:
Happy with this PR, Should we expand this to cover `utl` though?

Reply:
Good point. I guess we can start added this. I guess we'll need to do run a general update sometime soon anyways, but doesn't hurt to start. Will update

Reply:
done @ChrisSidebotham